service-front/module/Application/src/Controller/CPFlowController.php

```diff
@@ -5,25 +5,19 @@
 namespace Application\Controller;
 
 use Application\Contracts\OpgApiServiceInterface;
-use Application\Forms\DrivingLicenceNumber;
 use Application\Forms\LpaReferenceNumber;
-use Application\Forms\PassportNumber;
-use Application\Forms\PassportDate;
-use Application\Services\FormProcessorService;
-use Application\Services\SiriusApiService;
-use Application\Validators\LpaUidValidator;
+use Application\Helpers\FormProcessorHelper;
+use Laminas\Form\Annotation\AttributeBuilder;
 use Laminas\Http\Response;
 use Laminas\Mvc\Controller\AbstractActionController;
 use Laminas\View\Model\ViewModel;
-use Laminas\Form\Annotation\AttributeBuilder;
-use Application\Forms\NationalInsuranceNumber;
 
 class CPFlowController extends AbstractActionController
 {
     protected $plugins;
     public function __construct(
         private readonly OpgApiServiceInterface $opgApiService,
-        private readonly FormProcessorService $formProcessorService,
+        private readonly FormProcessorHelper $formProcessorHellper,
```

Comment:
Nit: spelling mistake on this property name
```suggestion
        private readonly FormProcessorHelper $formProcessorHelper,
```